### DISCUSSION TITLE
What is 'Use local DB' for?

### DISCUSSION THREAD

**Discussion by pto2k** [Comfy-Org/ComfyUI-Manager#26] [Q&A] [answered]
Readme does not explain much: 
When the 'Use local DB' feature is enabled, the application will utilize the data stored locally on your device, rather than retrieving node/model information over the internet

Just curious should a normal user use this option? How to mantain/sync the local data?

Thanks

**Comment by ltdrdata** [comment, marked as answer]
If it is checked then manager takes .json file from local instead of remote git repo.

Generally, the purpose is validation of updated db before upload.

The 'Use local DB' option is not necessary for regular end users. However, if you are registering a new custom node and submitting a pull request, it is recommended to use this option for testing purposes to ensure there are no issues.
